huxley/www/js/components/ChairAttendanceView.js

```diff
@@ -47,9 +77,10 @@ var ChairAttendanceView = React.createClass({
               <thead>
                 <tr>
                   <th>Assignment</th>
-                  <th>Present</th>
-                  <th>Present2</th>
-                  <th>Present3</th>
+                  <th>Friday</th>
```

Comment:
As I mentioned in the other PR, Session One, Session Two, Session Three, and Session Four might be more manageable names